src/coreclr/inc/jiteeversionguid.h

```diff
@@ -43,11 +43,11 @@ typedef const GUID *LPCGUID;
 #define GUID_DEFINED
 #endif // !GUID_DEFINED
 
-constexpr GUID JITEEVersionIdentifier = { /* e6c7a441-8d3c-4135-abc2-bc4dd7c6a4d7 */
-    0xe6c7a441,
-    0x8d3c,
-    0x4135,
-    {0xab, 0xc2, 0xbc, 0x4d, 0xd7, 0xc6, 0xa4, 0xd7}
+constexpr GUID JITEEVersionIdentifier = { /* 09a675fa-5035-4437-ae49-824fa63aa69a */
```

Comment:
Why do we need to rev the GUID? This change is not modifying the JIT/EE interface contract.

(It certainly does not hurt to rev the GUID, just wondering why it is necessary.)

Reply:
Reverted, initially I also wanted to test 1ES change for SPMI along the way, but it's already checked